Altis_Life.Altis/config/Config_SpyGlass.hpp

```diff
@@ -312,17 +312,17 @@ class SpyGlass {
         { "bis_egspectator_showheader", "BOOL" }, { "cash", "SCALAR" }, { "life_action_spikestripdeploy", "SCALAR" }, { "rscegspectator_lastticktime", "SCALAR" }, { "bis_pp_burndamage", "ARRAY" }, { "life_safeobj", "OBJECT" }, { "life_dp_point", "OBJECT" }, { "life_dp_start", "OBJECT" }, { "bis_fnc_diagmacrossimpleobject", "CODE" }, { "bis_fnc_groupfromnetid", "CODE" }, { "bis_fnc_tasktype", "CODE" },
         { "bis_fnc_tasksetalwaysvisible", "CODE" }, { "bis_fnc_tasktypeicon", "CODE" }, { "bis_fnc_tasksettype", "CODE" }, { "bis_fnc_objectfromnetid", "CODE" }, { "bis_fnc_sunrisesunsettime", "CODE" }, { "bis_fnc_netid", "CODE" }, { "bis_fnc_getarea", "CODE" }, { "bis_fnc_taskalwaysvisible", "CODE" }, { "life_markers_vehicles", "ARRAY" }, { "life_markers_players", "ARRAY" },
         { "advanced_log", "STRING" }, { "money_log", "STRING" }, { "life_markers_Vehicles", "ARRAY" }, { "life_markers_Players", "ARRAY" }, { "life_vinact_curtarget", "OBJECT" }, { "life_ticket_cop", "OBJECT" }, { "life_ticket_unit", "OBJECT" }, { "life_ticket_val", "SCALAR" }, { "life_ticket_paid", "BOOL" }, { "life_boltcutter_uses", "SCALAR" }, { "life_pinact_container", "OBJECT" },
         { "luce_1", "SCRIPT" }, { "life_bail_amount", "SCALAR" }, { "life_canpay_bail", "BOOL" }, { "hc_life", "SCALAR" }, { "life_fnc_requestclientid", "OBJECT" }, { "life_hc_isactive", "BOOL" }, { "ton_fnc_playtime_values", "ARRAY" }, { "ton_fnc_playtime_values_request", "ARRAY" }, { "hc_1", "OBJECT" }, { "hc_life", "BOOL" }, { "rscdisplaympinterrupt_respawntime", "SCALAR" }, { "bis_dynamictext_spawn_8", "SCRIPT" },
-        { "life_my_gang", "OBJECT" }, {"am_exit","SCALAR"}, { "life_garage_sp", "ARRAY" }, { "life_garage_sp", "STRING" }, { "0", "ARRAY" }, { "life_oldvestitems", "ARRAY" }, { "life_shop_cam", "OBJECT" }, { "life_oldclothes", "STRING" }, { "life_cmenu_lock", "BOOL" }, { "life_oldhat", "STRING" }, { "life_oldvest", "STRING" }, { "life_oldglasses", "STRING" }, { "life_oldbackpackitems","ARRAY"},
-        {"life_oldbackpack","STRING"}, { "rscnotification_data", "ARRAY" }, { "life_curwep_h", "STRING" }, { "carshop_lux_1", "OBJECT" }, { "life_olduniformitems", "ARRAY" }, { "bis_fnc_setvehiclemass_fsm", "SCALAR" }, { "life_3dPreview_light", "OBJECT" }, { "life_3dPreview_camera", "OBJECT" }, { "life_3dPreview_object", "OBJECT" },
+        { "life_my_gang", "OBJECT" }, { "am_exit}, {SCALAR" }, { "life_garage_sp", "ARRAY" }, { "life_garage_sp", "STRING" }, { "0", "ARRAY" }, { "life_oldvestitems", "ARRAY" }, { "life_shop_cam", "OBJECT" }, { "life_oldclothes", "STRING" }, { "life_cmenu_lock", "BOOL" }, { "life_oldhat", "STRING" }, { "life_oldvest", "STRING" }, { "life_oldglasses", "STRING" }, { "life_oldbackpackitems", "ARRAY" },
```

Comment:
am_exit and SCALAR have been seperated?

Reply:
that is just a string of `"am_exit}, {SCALAR"`